mta/test_framework/pytest_plugin.py

```diff
@@ -0,0 +1,12 @@
+import logging
+
+logger = logging.getLogger("mta.test_framework.pytest_plugin")
```

Comment:
We are not using logger in this this file.

Reply:
ahhh It was old stuff... Understand instate of renaming you deleted this file thats why github showing it. :+1: